app/code/Magento/InventoryCatalog/Api/UpdateLegacyCatalogInventoryStockItemByPlainQueryInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryCatalog\Api;
+
+use Magento\CatalogInventory\Api\Data\StockItemInterface;
+
+/**
+ * Update Legacy catalocinventory_stock_item database data
+ *
+ * @api
+ */
+interface UpdateLegacyCatalogInventoryStockItemByPlainQueryInterface
```

Comment:
we no need to introduce API for  Updating LegacyCatalogInventoryStockItem
as this is not a kind of business operation which would be called directly in application logic.
This is something which would be called under the hood, internally in the scope of other API invocation. 

Moreover, after Milestone 2 of MSI would be released - we will remove integration with legacy system.

The same true for `UpdateLegacyCatalogInventoryStockItemByPlainQueryInterface`

Actually, it should be a sign that we don't use these interfaces directly, but in plugin only (which injects/intercepts another Business Operation). So, this is not an independent business operation which should have own API